prow/config/config.yaml

```diff
@@ -644,7 +679,7 @@ presubmits:
     - name: build
       branches:
         - main
-      run_if_changed: '^api|^test|^Makefile$'
+      run_if_changed: "^api|^test|^Makefile$"
```

Comment:
i see single quotes being used consistently in the file though

Reply:
This is the same exact auto-formatting issue I had in my PR. I'm fairly sure that the double-quote is the de-facto formatting we should be using on all lines, but for whatever reason it only triggers the formatting on this one line. Anyways, revert to single-quote as @furkatgofurov7 suggested.

Reply:
Yeah it was the formatter. Updated it.